code/modules/reagents/withdrawal/generic_addictions.dm

```diff
@@ -154,3 +154,46 @@
 	REMOVE_TRAIT(affected_human, TRAIT_NIGHT_VISION, type)
 	var/obj/item/organ/eyes/eyes = affected_human.getorgan(/obj/item/organ/eyes)
 	eyes.refresh()
+
+///Makes you a hypochondriac - I'd like to call it hypochondria, but "I could use some hypochondria" doesn't work
+/datum/addiction/medicine
+	name = "medicine"
+	var/datum/hallucination/fake_alert/hallucin
+
+/datum/addiction/medicine/withdrawal_stage_1_process(mob/living/carbon/affected_carbon, delta_time)
+	. = ..()
+	if(prob(10*delta_time))
+		affected_carbon.emote("cough")
+
+/datum/addiction/medicine/withdrawal_enters_stage_1(mob/living/carbon/affected_carbon)
+	. = ..()
+	var/type = pick(list("not_enough_oxy", "too_much_oxy", "temphot", "tempcold"))
+	hallucin = New(affected_carbon, TRUE, type)
+
+/datum/addiction/medicine/withdrawal_enters_stage_2(mob/living/carbon/affected_carbon)
+	. = ..()
+	affected_carbon.hal_screwyhud = SCREWYHUD_CRIT
```

Comment:
is there a screwyhud for being at low health? that might be more believable, especially if accompanied by fake injuries on your health doll and/or a movement speed penalty

Reply:
I can look into it, but I don't think there was. Maybe I can write my own!

Reply:
I wrote my own! It should give you fake damage!